src/main/fc/config.c

```diff
@@ -1096,7 +1092,7 @@ void validateAndFixGyroConfig(void)
     float motorUpdateRestriction;
     switch(motorConfig()->motorPwmProtocol) {
         case (PWM_TYPE_STANDARD):
-            motorUpdateRestriction = 0.0025f;
+            motorUpdateRestriction = 0.002f;
```

Comment:
This will not properly reset PWM since it sets `motor_update_rate` to 500 if it's higher.